samples/python/llm/utils/pt_utils.py

```diff
@@ -0,0 +1,221 @@
+            else:
+                print(f"load image model from model path:{model_path}")
+                model_type = DEFAULT_MODEL_CLASSES[kwargs['use_case']]
+                model_class = PT_MODEL_CLASSES_MAPPING[model_type]
+                start = time.perf_counter()
+                pipe = model_class.from_pretrained(model_path)
+                end = time.perf_counter()
+                from_pretrain_time = end - start
+        else:
+            raise RuntimeError(f"==Failure ==: model path:{model_path} is not directory")
+    else:
+        raise RuntimeError(f"==Failure ==: model path:{model_path} is not exist")
+    
+    print(f"model path:{model_path}, from pretrained time: {from_pretrain_time:.2f}s")
+
+    if device is not None:
+        pipe.to(device)
+    else:
+        raise RuntimeError("==Failure ==: no device to load")
+    return pipe, from_pretrain_time
+
+def create_image_classification_model(model_path, device, **kwargs):
+    model_path = Path(model_path)
+    model_file = None
+    model_id = None
+    if model_path.exists():
+        if model_path.is_dir():
+            model_file = list(model_path.glob("*.pth"))
+            if model_file:
+                model_file = model_file[0]
+            else:
+                model_file = None
+            model_id = model_path.name
+        else:
+            model_file = model_path
+            model_id = model_path.name.replace(".pth", "")
+    else:
+        model_id = model_path.name.replace(".pth", "")
+    model = timm.create_model(model_id, pretrained=model_file is None)
+    if model_file:
+        model.load_state_dict(torch.load(model_file, map_location="cpu"))
+    else:
+        print(model.state_dict())
+        torch.save(model.state_dict(), model_path / f"{model_id}.pth")
+    if device:
+        model.to(device)
+    else:
+        raise RuntimeError("==Failure ==: no device to load")
+    model.eval()
+    data_config = timm.data.resolve_data_config([], model=model_id, use_test_size=True)
+    input_size = (1, ) + data_config["input_size"]
+    return model, input_size
+
+def create_ldm_super_resolution_model(model_path, device, **kwargs):
+    model_path = Path(model_path)
+    from_pretrain_time = 0
+    if model_path.exists():
+        if model_path.is_dir():
+            # Checking if the list is empty or not
+            if len(os.listdir(model_path)) == 0:
+                if kwargs['model_id'] != "":
+                    print(f"get super resolution model from huggingface...")
+                    connect_times = 1
+                    pipe, from_pretrain_time = get_image_model_from_huggingface(model_path, connect_times, **kwargs)
+                else:
+                    raise RuntimeError(f"==Failure ==: the model id of huggingface should not be empty!")
+            else:
+                print(f"load image model from model path:{model_path}")
```

Comment:
Please capitalize the first character in the print sentence.

Reply:
Resolved, thanks.